src/LogViewer/chunk-worker.js

```diff
@@ -128,14 +128,21 @@ const update = (response) => {
 const error = () => self.postMessage(JSON.stringify({ type: 'error' }));
 const loadEnd = () => self.postMessage(JSON.stringify({ type: 'loadend' }));
 
-const toHtml = (index) => {
+const getParagraphClass = (lineNumber, metadata) => {
+  const { highlightStart, highlightEnd } = metadata;
+
+  return lineNumber >= highlightStart && lineNumber <= highlightEnd ? 'highlight' : '';
+};
+
+const toHtml = (index, metadata) => {
   const lines = decode(chunks[index]);
   const start = LINE_CHUNK * index + offset + 1;
 
   const html = lines.map((parts, index) => {
     const lineNumber = start + index;
+    const pClass = getParagraphClass(lineNumber, metadata);
 
-    return `<p><a id="${lineNumber}">${lineNumber}</a>${parts.map((part) => {
+    return `<p class="${pClass}"><a id="${lineNumber}">${lineNumber}</a>${parts.map((part) => {
```

Comment:
Since we are potentially dealing with a lot of data, I want to try and reduce the length of the strings we send as much as possible. Because of that, `class` could potentially be an empty string, so lets make the `getParagraphClass` take care of that. I'll comment there for the change to the function, and how the `p` should look:

```js
return `<p${pClass}><a ...`
```